WebServer/server_first.py

```diff
@@ -0,0 +1,49 @@
+from http.server import BaseHTTPRequestHandler, HTTPServer
+from http.cookies import SimpleCookie
+
+from read_page_inf import read_page
+from error_handler import error_page
+
+
+class ServerFirst(BaseHTTPRequestHandler):
```

Comment:
Very pedantic thing, but it should be either `FirstServer` or `ServerOne`, same for `ServerSecond`

Reply:
Corrected